mm2src/coins/Cargo.toml

```diff
@@ -44,9 +44,9 @@ derive_more = "0.99"
 ed25519-dalek = "1.0.1"
 enum_derives = { path = "../derives/enum_derives" }
 ethabi = { version = "17.0.0" }
-ethcore-transaction = { git = "https://github.com/KomodoPlatform/mm2-parity-ethereum.git" }
+ethcore-transaction = { git = "https://github.com/dimxy/mm2-parity-ethereum.git", branch = "eip1559-support" }
 ethereum-types = { version = "0.13", default-features = false, features = ["std", "serialize"] }
-ethkey = { git = "https://github.com/KomodoPlatform/mm2-parity-ethereum.git" }
+ethkey = { git = "https://github.com/dimxy/mm2-parity-ethereum.git", branch = "eip1559-support" }
```

Comment:
This PR https://github.com/KomodoPlatform/mm2-parity-ethereum/pull/1 is merged, you can use the original branch again.

Reply:
fixed in [342d3f4](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2051/commits/342d3f4b8cfc50b9c4310ca262bd3314de0d2e9d)